spec/features/happy_journeys_spec.rb

```diff
@@ -227,7 +227,7 @@
     expect(check_answers_page.summary_list["Full name"].value).to eql("John Doe")
     expect(check_answers_page.summary_list["TRN"].value).to eql("1234567890")
     expect(check_answers_page.summary_list["Date of birth"].value).to eql("December 13, 1980")
-    expect(check_answers_page.summary_list["National Insurance number"].value).to eql("AB123456C")
+    expect(check_answers_page.summary_list["National Insurance Number"].value).to eql("AB123456C")
```

Comment:
`Number` should be lower case see https://design-system.service.gov.uk/patterns/national-insurance-numbers/